offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -749,6 +748,18 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
     }
   }
 
+  @Override
+  Optional<Integer> findCustomer(Customer customer) {
+    int customerID
+
+    findActiveCustomer(customer.firstName, customer.lastName).each {foundCustomer ->
+      //todo is the email address sufficient to compare customers for identity?
```

Comment:
```suggestion
      //todo is the email address sufficient to compare customers for identity? We might want to use user_id later.
```

Reply:
hm not sure if we should state that here. I think we should rather document it as an issue to better track it